### PR TITLE
Disable lux renderer from backend

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

**Review by Krigpl — APPROVED** [review]
thisisgine

File: setup.py [hunk 1]
```diff
@@ -65,6 +65,7 @@
         ]),
         (path.normpath('../../golem/apps'), [
             path.normpath('apps/registered.ini'),
+            path.normpath('apps/registered_test.ini'),
```
Comment (maaktweluit) [inline setup.py:68]: I think a similar line should also be added to the pyinstaller hooks to bundle this new file in the binaries

Comment (mfranciszkiewicz) [reply setup.py:68]: @maaktweluit We're using a wildcard for `ini` files in that directory.

Comment (maaktweluit) [reply setup.py:68]: Thanks TIL :D

**Pull request merged**